operating_unit/models/res_users.py

```diff
@@ -44,6 +46,21 @@ def _default_operating_units(self):
         default=lambda self: self._default_operating_unit(),
     )
 
+    def write(self, vals):
+        res = super(ResUsers, self).write(vals)
+        if any(
+            user.default_operating_unit_id
+            and user.default_operating_unit_id not in user.operating_unit_ids
+            for user in self
+        ):
+            raise ValidationError(
+                _(
+                    """The chosen Operating Unit is not in
```

Comment:
please use single quotes, otherwise the message does not display correctly

Reply:
I have removed this code and made the validation through XML itself.